### ISSUE TITLE
[0.5.0] White flash when closing tab using X in TSR sidebar

### ISSUE THREAD

**Issue by SXZ1** [eoger/tabcenter-redux#222]
TSR0.5.0, Nightly 22-10-17. STR:
Create new profile, install TSR, open new window, go to https://forums.geforce.com/default/board/110/ in the active tab
Open new tab, go to https://forums.geforce.com/default/board/27/support/
Press Ctrl-1, hover the mouse over the first tab in the sidebar and click X to close it

ER: Tab is closed, no white flash
AR: Tab is closed, white flash
Bug is not reproduced when using Ctrl-W or clicking X in the Firefox tab bar. This bug can also be reproduced multiple times without Firefox restart: just press Ctrl-Shift-T and try to close the tab again.

I think this bug might be related to https://bugzilla.mozilla.org/show_bug.cgi?id=1237654
Long story short: toolkit.cosmeticAnimations.enabled false causes white flash when closing tabs in stock Firefox/Nightly

**Comment by eoger** [comment]
I have commented on the bug, thanks

**Comment by SXZ1** [comment]
It looks like the solution is to trigger tab closing animation in Firefox tab bar after pressing X in TCR sidebar. I don't know if that's possible, though.

**Comment by eoger** [comment]
Closing this since it's a Firefox bug, I'll update this issue if they fix it.
